schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,53 @@
+title: MediaWiki Page Delete
```

Comment:
After an event of this type, is it ok to still have the historic revisions of this page published?